src/WooCommerce.php

```diff
@@ -480,8 +480,12 @@ public static function woocommerce_edit_account_form() {
         'priority' => 130,
       ],
     ];
+    $userinfo = Plugin::buildUserInfo('account', get_current_user_ID());
 
     foreach ($opt_ins as $opt_in_id => $opt_in_args) {
+      if ($userinfo['optins'][$opt_in_id]) {
+        continue;
```

Comment:
> Do not display opt-in for general terms & conditions (confirm_agb) once confirmed, but ensure to retain value in updates

This behavior is only supposed to be applied to the terms & conditions opt-in, because the user has to agree in order to purchase/sign-up.

Reply:
Yep, got it now, after our recent Slack discussion. Thanks!